tests/acceptance/05_processes/02_functions/findprocesses.cf

```diff
@@ -2,6 +2,9 @@
 #
 # Test findprocesses()
 #
+# Note: On HP-UX this depends on the
+# 00_hpux_ps_setup_serial.cf setup test having run first.
```

Comment:
Why does it need to run first? Maybe edit the comment and/or the commit message?